### ISSUE TITLE
How plugin updates are determined ?

### ISSUE THREAD

**Issue by glhez** [mojohaus/versions-maven-plugin#214]
I created this issue because I've  encountered  a problem  on  my Windows workstation, where the display-plugin-updates was not showing the latest plugin version. As an example using a [simple pom](https://github.com/mojohaus/versions-maven-plugin/files/1347084/pom.txt), the following plugins are updated on central and are not shown (I added the latest version to the plugin output):

```
$ mvn org.codehaus.mojo:versions-maven-plugin:2.5:display-plugin-updates
[INFO] Require Maven 3.0 to use the following plugin updates:
[INFO]   maven-assembly-plugin ......................................... 3.0.0 -> 3.1.0
[INFO]   maven-compiler-plugin ......................................... 3.6.1 -> 3.7.0
[INFO]   maven-dependency-plugin ....................................... 3.0.1 -> 3.0.2
[INFO]   maven-javadoc-plugin ......................................... 2.10.4 -> 3.0.0-M1
[INFO]   org.codehaus.mojo:versions-maven-plugin ......................... 2.3 -> 2.5 !
```

After clearing the whole `%USERPROFILE%\.m2\repository` and forcing maven to redownload all stuff, the versions plugin was finally showing the updates.

What is the deciding factor in showing a "new version" ? 
Could the version plugin not check online for plugin updates ?

**Comment by slachiewicz** [comment]
This depends on "update policy" for repository

You can force update (without clearing .m2\repository) with mvn -U switch

**Comment by glhez** [comment]
I'll try the -U if I remember it for the next time :)
That's still strange: I did not touch the `updatePolicy` locally (I learned about it after your answer) and the doc says [it is `daily` by default](https://maven.apache.org/ref/3.5.2/maven-settings/settings.html#class_releases); I presume there is some exception for central but yet, I remember having seen an update of `maven-compiler-plugin` without the need for `-U` or purging the repository.
That question aside, I'm now using bintray as well as mirroring to it. I close my question waiting for the problem to (never?) re-appear.
